src/main/java/com/revlearn/team1/dto/discussionBoard/DiscussionBoardFullDTO.java

```diff
@@ -0,0 +1,9 @@
+package com.revlearn.team1.dto.discussionBoard;
```

Comment:
what are the Full vs Light DTOs supposed to represent? Are they both API requests or responses?